demisto_sdk/commands/upload/tests/uploader_test.py

```diff
@@ -0,0 +1,18 @@
+from unittest.mock import patch
+
+from demisto_sdk.commands.common.git_tools import git_path
+from demisto_sdk.commands.upload.uploader import Uploader
+
+
+@patch('demisto_client.configure')
+def test_upload(mocked_configure=None):
```

Comment:
separate into two steps with indicative names:
`test_upload_sanity`
`test_upload_invalid_server` or `test_upload_failure`

optimal test: checking that an error is printed
you can try to mock `print_color` and check that the failure actually calls that function. I tried to in the past and it didn't work so I wouldn't have wasted time on it...